integration-tests/suites/base.go

```diff
@@ -390,24 +390,13 @@ func (s *IntegrationTestSuiteBase) waitForFileToBeDeleted(file string) error {
 			}
 		}
 	}
-
-	return nil
 }
 
 // isSelinuxPermissiveNeeded returns whether or not a given VM requires
 // SELinux permissive mode. e.g. rhel, or fedora-coreos
 func (s *IntegrationTestSuiteBase) isSelinuxPermissiveNeeded() bool {
 	vmType := config.VMInfo().Config
-	if strings.Contains(vmType, "coreos") || strings.Contains(vmType, "rhcos") {
```

Comment:
Not sure how this change made it in, I think I hadn't pushed the base branch when I created this branch. I'll juggle commits at some point, or we can just merge this change as part of this PR (I don't think it strictly affects either of them)

Reply:
I think it's fine, we can merge it as part of this PR.